pom.xml

```diff
@@ -6,13 +6,13 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.40</version>
+        <version>4.49</version>
         <relativePath />
     </parent>
 
     <groupId>org.biouno</groupId>
     <artifactId>uno-choice</artifactId>
-    <version>2.6.5-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
```

Comment:
The result of applying this [best practice](https://www.jenkins.io/doc/developer/tutorial-improve/enable-incrementals/).